pkg/apis/batch/v1alpha1/job.go

```diff
@@ -97,6 +97,10 @@ type JobSpec struct {
 	// If specified, indicates the job's priority.
 	// +optional
 	PriorityClassName string `json:"priorityClassName,omitempty" protobuf:"bytes,10,opt,name=priorityClassName"`
+
+	// The minimal success pods to run for this Job
+	// +optional
+	MinSuccess *int32 `json:"minSuccess,omitempty" protobuf:"varint,11,opt,name=minSuccess"`
```

Comment:
Why set this field as *int , not int?

Reply:
I still think this field should be *int. 

`MinSuccess` is an **optional** feature, If the user does not set this field, the logic should same as the original.